appveyor.yml

```diff
@@ -15,8 +15,9 @@ test_script:
   - node --version
   - npm --version
   # run tests
-  - yarn test
-  - yarn fmt-check
+  - yarn run lint
+  - yarn run build
```

Comment:
Nope, Appveyor documentation https://www.appveyor.com/docs/lang/nodejs-iojs/ suggests doing it this way